Policies/github-corporate-terms-of-service.md

```diff
@@ -132,45 +133,58 @@ You agree not to engage in activity that significantly harms our Users. We will
+If you set your pages and repositories to be viewed publicly, you grant each user of GitHub a nonexclusive, worldwide license to use, display, and perform Your Content through the GitHub Service and to reproduce Your Content solely on GitHub as permitted through GitHub's functionality (for example, through forking). You may grant further rights if you [adopt a license](/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository). If you are uploading Content you did not create or own, you are responsible for ensuring that the Content you upload is licensed under terms that grant these permissions to other GitHub users.
 
 #### 6. Contributions Under Repository License
-Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supercede.
+Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supersede.
 
 Isn't this just how it works already? Yep. This is widely accepted as the norm in the open-source community; it's commonly referred to by the shorthand "inbound=outbound". We're just making it explicit.
 
 #### 7. Moral Rights
-You retain all moral rights to Content you upload, publish, or submit to any part of the Service, including the rights of integrity and attribution. However, you waive these rights and agree not to assert them against us, to enable us to reasonably exercise the rights granted in Section D.4, but not otherwise. You understand that you will not receive any payment for any of the rights granted in this Section.
+You retain all moral rights to Your Content that you or your Users upload, publish, or submit to any part of the Service, including the rights of integrity and attribution. However, you waive these rights and agree not to assert them against us, to enable us to reasonably exercise the rights granted in Section D.4, but not otherwise.
+
+To the extent this agreement is not enforceable by applicable law, you grant GitHub the rights we need to use Your Content without attribution and to make reasonable adaptations of Your Content as necessary to render the Website and provide the Service.
```

Comment:
@Oneiroi Thanks for the suggestion, but it would overcomplicate our Terms if we wrote them to anticipate every possible situation. 

@cosimo Good question. You grant GitHub the rights we need (to use Your Content without attribution AND to make reasonable adaptations of Your Content) as necessary to render the Website and provide the Service.

"...as necessary to render the Website and provide the Service" limits both.